### DISCUSSION TITLE
Cannot print Chinese characters properly in VScode R notebook

### DISCUSSION THREAD

**Discussion by ferris-wufei** [microsoft/vscode-jupyter#9738] [Questions and Answers]
```text
VScode version
Version: 1.66.2 (Universal)
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:20.994Z (1 wk ago)
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Darwin x64 21.2.0
---
Jupyter extension version
v2022.4.1000912146
---
R version 4.1.3 (2022-03-10) -- "One Push-Up"
```

Hi,

I'm using Jupyter notebook with R kernel in VScode. The Chinese characters cannot be displayed correctly. Seems they are converted to unicodes. Meanwhile everything works good in R terminal. 

Here're some examples.

```text
 cat('你好')
> <U+4F60><U+597D>

'你' %in% c('你', '好')
> TRUE
``` 

This caused bigger issue such as the SQL commands with Chinese are not properly sent to DB connections...

Am I missing any config settings? Appreciate your help :)

**Comment by DonJayamanne** [comment]
This seems to work for me, 
Please could you provide some screenshots of the problem

Here's a screenshot of it working for me
![image](https://user-images.githubusercontent.com/1948812/165134251-d3e0ff0f-b170-4aca-ad7a-aff000633fe3.png)

**Reply by ferris-wufei** [reply]
<img width="614" alt="image" src="https://user-images.githubusercontent.com/15623440/165191068-ca9446a4-7706-4df7-abd4-2f7012540390.png">

meanwhile this doesn't happen in R terminal

<img width="508" alt="image" src="https://user-images.githubusercontent.com/15623440/165191264-6a16ed10-40fd-430b-90f9-122d601b2887.png">

Here's the VScode settings

```
{
    "workbench.colorTheme": "GitHub Light Default",
    "editor.fontSize": 13,
    "r.bracketedPaste": true
}
```

System locale

```
> locale
LANG="zh_CN.UTF-8"
LC_COLLATE="zh_CN.UTF-8"
LC_CTYPE="zh_CN.UTF-8"
LC_MESSAGES="zh_CN.UTF-8"
LC_MONETARY="zh_CN.UTF-8"
LC_NUMERIC="zh_CN.UTF-8"
LC_TIME="zh_CN.UTF-8"
LC_ALL="zh_CN.UTF-8"
```

I had to switch back to in-browser Jupyter which works fine.

**Reply by DonJayamanne** [reply]
I've created an issue here https://github.com/microsoft/vscode-jupyter/issues/9777
Lets continue the discussion there (as this looks like a bug), please provide the information requested in the issue.
